features/admin.identity-providers.v1/configs/ui.ts

```diff
@@ -96,6 +96,7 @@ export const getAuthenticatorIcons = (): any => {
         microsoft: MicrosoftLogo,
         office365: Office365Logo,
         push: PushIcon,
+        "push-notification-authenticator": PushIcon,
```

Comment:
Cannot we reuse the icon defined in L98?

Reply:
It is the same icon, but in different places, the same icons has to be loaded with different key values. Similar keys and values are already available

Reply:
I see, thanks